templates/guide.html

```diff
@@ -38,15 +38,15 @@
           <div class="web-author-information__details">
             <span class="web-author-information__written-by">Written by</span>
             <span>{{author.name.given}} {{author.name.family}}</span>
-            <div class="web-author-information__links"></div>
+            <div class="web-author-information__links">
               {{#if author.glitch}}
-              <a class="web-author-information__link" href="{{author.glitch}}">Glitch</a>
+              <div class="web-author-information__link"><a href="{{author.glitch}}">Glitch</a></div>
```

Comment:
Do you have a screenshot of the before and after? Just curious why you needed the extra div.

Reply:
When hovering over a link, the dot would also receive a big (ugly) underline. This was because the link was display flex. Not making it flex meant that the text was not centered. The eventual solution was to make a wrapping div which is flex and give that the dot.